app/community/seminar/page.tsx

```diff
@@ -11,89 +11,55 @@ import SeminarRow from '@/components/seminar/SeminarRow';
 
 import { useCustomSearchParams } from '@/hooks/useCustomSearchParams';
 
-import { seminar } from '@/types/page';
-import { GETSeminarPostsResponse, SimpleSeminarPost } from '@/types/post';
-
 const postsCountPerPage = 10;
 
-interface SeminarListWithYear extends SimpleSeminarPost {
-  year?: number;
-}
-
 export default function SeminarPage() {
   const { page, keyword, setSearchParams } = useCustomSearchParams();
-  const [totalPostsCount, setTotalPostsCount] = useState(0);
-  const [posts, setPosts] = useState<SeminarListWithYear[]>([]);
+
   const setCurrentPage = (pageNum: number) => {
     setSearchParams({ purpose: 'navigation', page: pageNum });
   };
 
-  const fetchPost = useCallback(async () => {
-    const res = await getMockSeminarPosts({
-      keyword: keyword === null ? undefined : keyword,
-      page: page,
-    });
-
-    const postsWithYear: SeminarListWithYear[] = [];
-
-    res.seminarList.forEach((post) => {
-      const postYear = new Date(post.date).getFullYear();
-
-      if (post.isLast) {
-        postsWithYear.push({
-          ...post,
-          year: postYear,
-        });
-      } else {
-        postsWithYear.push({
-          ...post,
-        });
-      }
-    });
-
-    setTotalPostsCount(res.total);
-    setPosts(postsWithYear);
-  }, [keyword, page]);
-
-  useEffect(() => {
-    fetchPost();
-  }, [fetchPost]);
+  const { data, isLoading, error } = useSWR([keyword, page], getMockSeminarPosts);
```

Comment:
이거 아마 이렇게 해야 될 거예요! `/app/swr-provider.tsx`에 설정해둔 거 참고...!
```suggestion
  const { data, isLoading, error } = useSWR({url: "/seminar", params: { keyword, page } }, getMockSeminarPosts);
```